src/main/java/com/hyperwallet/clientsdk/model/HyperwalletDocumentRejectReason.java

```diff
@@ -1,16 +1,31 @@
 package com.hyperwallet.clientsdk.model;
 
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.hyperwallet.clientsdk.util.HyperwalletJsonConfiguration;
+
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlRootElement;
 
+@JsonFilter(HyperwalletJsonConfiguration.INCLUSION_FILTER)
 @XmlRootElement
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletDocumentRejectReason {
 
     private DocumentVerificationReason name;
     private String description;
 
+    public enum DocumentVerificationReason {
```

Comment:
Is there specific reason to put the enum within this class? Usually we keep enums in their own class. It makes referencing them outside of this class easier.

Reply:
I was thinking to put it in it's own file, but in the existing HyperwalletUser.java class, all the enums are created within the class,  tried to follow the same pattern. And for referencing them outside of this class, it's public and can be referenced. It's being used in HyperwalletTest.java

Reply:
Ok if that is how they did for other enums, we'll leave it. That's fine.